frameworks/person-escort-record/README.md

```diff
@@ -74,8 +74,11 @@ Question files should live in [`questions/`](./questions) and be stored in a fla
   - `followup_comment` - allow for a text box input to be displayed conditionally if a specific value from a question has been selected
     - `label` **(required)** - text displayed on the text box label
     - `validations` - list of [validations](#validations) that need to be applied to this text box
-      - `name` **(required)** - the validation error key
+      - `type` **(required)** - the validation error key
       - `message` - custom text that will be displayed in the form
+  - `flags` - list of flags to conditionally display on summary pages as alerts if this question is answered with this value.
+    - `key` **(required)** - the flag key
```

Comment:
Do you think we need the `key` in the manifest? I wonder if it requires another thing that needs to be correctly spelled, etc and needs to match other uses of it.

I was also having a look through the designs again last night and spotted a flag for "special vehicle" that used a different style to the other flags in that category.

So I think we may need to include the ability to determine the flag type or category when defining it here. I wonder if this should become:

```yaml
flags:
  -
    type: risk # although not 100% how we should categorise these - risk/health/???
    label: Flag name
```

Reply:
discussed in person but for posterity: remove `key` field as we don't really need it and adds maintenance overhead. Add `type` field as we need to define flags as different colours on the frontend. So far we have 4 types: 
`info` - ❔  grey
`attention` - 🔵  blue
`warning` - 🟡 yellow
`alert` - 🔴  red